articles/active-directory/manage-apps/methods-for-assigning-users-and-groups.md

```diff
@@ -84,7 +84,7 @@ After a short period of time, the users you have selected will be able to launch
 
 To assign one or more groups to an application directly, follow the steps below:
 
-1.  Open the [**Azure portal**](https://portal.azure.com/) and sign in as a **Global Administrator.**
+1.  Open the [**Azure portal**](https://portal.azure.com/) and sign in as a **Global Administrator or as a non-admin application owner with an .**
```

Comment:
Please complete this -- I presume it's the same as above?

Reply:
corrected statement describing the behavior for assigning a group as well as the statement for assigning a user.